### ISSUE TITLE
Remember me for the login page

### ISSUE THREAD

**Issue by alanmilinovic** [OliveTin/OliveTin#446]
Please add "remember me" checkbox on login page so that we don't have to login every time. Best option would be with file based storage to keep the session persistence.

**Comment by jamesread** [comment]
I changed the cookies for Local User and OAuth2 to last for 1 year (previously they were until the tab was closed). This should help quite a bit.

**Comment by alanmilinovic** [comment]
I would rather put a shorte period and extend it if user is active. Other thing is persistance across reboots and restarts, that would be good to have in place.

**Comment by alanmilinovic** [comment]
If I close the browser, session is lost. Just tested.

**Comment by alanmilinovic** [comment]
Any chance enhancing login persistence across reboot?

**Comment by jamesread** [comment]
Yep, I'll add this shortly - sorry I've not been very responsive here - working on other projects and lots of work recently. 

I'll make a persistent cookie file and include that in the next release.

**Comment by alanmilinovic** [comment]
Implemented and tested. Thank you!
